include/deal.II/fe/fe_base.h

```diff
@@ -242,6 +242,13 @@ class FiniteElementData
    */
   const unsigned int dofs_per_hex;
 
+  /**
+   * Number of non-local degrees of freedom on a cell. This is the
+   * number of degrees of freedom support the finite element but can not be
```

Comment:
how about:

This is the number of non-zero degrees of freedom (DoFs) on the cell that, however, can not be associated with the specific geometric object (such as face or edge). In these cases, DoFs are typically globally distributed over (patches of) the Triangulation. Typical examples are IGA, XFEM, Catmull Clarks FE etc.